spec/units/image_spec.rb

```diff
@@ -23,7 +23,7 @@
     let(:file_name) { 'thumbnail' }
 
     it "should use the specified size and name" do
-      expect(subject).to receive(:create_resized_image).with(output_file, "200x300>", 'png')
+      expect(subject).to receive(:create_resized_image).with(file_name,"200x300>", 'png')
```

Comment:
Please add space after comma to conform to style guide: https://github.com/styleguide/ruby